docs/BuildFromSource.md

```diff
@@ -266,7 +268,7 @@ source ./activate.sh
 
 ### Running tests on command-line
 
-Tests are not run by default. Use the `-test` option to run tests in addition to building.
+Tests are not run by default. Within an project's parent directory, use the `-test` option to run tests in addition to building.
```

Comment:
TBH I find it a bit confusing phrased as-is, because it's not saying what you're using `-test` with.

```suggestion
Tests are not run by default. When invoking a `build.cmd`/`build.sh` script, use the `-test` option to run tests in addition to building.
```